### ISSUE TITLE
Option for selecting container inside pod

### ISSUE THREAD

**Issue by galan** [astefanutti/kubebox#13]
Having multiple containers running inside a pod, I need a selection mechanism to choose the pod for logs and executing into it.

**Comment by astefanutti** [comment]
You can select the container by just pressing ENTER. That enables rotating the pod containers.

That being said, That is a short term solution as it’s not intuitive. I was thinking to have a tree like pod / container list to make things more obvious to the user. It would solve that issue for remote exec into pod containers as well.

**Comment by astefanutti** [comment]
Just let me keep that one open, as this is an issue for container remote exec.

**Comment by ianseyer** [comment]
yes - even after using enter to rotate to the correct pod, pressing `r` still opens to the default container

**Comment by pnull** [comment]
👍

**Comment by astefanutti** [comment]
As a poor-man solution, 60184b1 uses the same container selection mechanism than the log widget, that is pressing ENTER on a pod to rotate over its containers. It'll be available in the next release.

Let's keep that issue open to track progress on improving the overall user experience for container selection, and navigation to other resources in general.

**Comment by ensonic** [comment]
+1 for the tree proposal. Using the 'enter' key is not discoverable right now. With a tree one could also select the pod to see all the the pods logs at once.

**Comment by astefanutti** [comment]
@ensonic thanks for the feedback. A tree would also be useful to arrange resources by ownership.
